compiler/ksp_compiler.py

```diff
@@ -554,9 +555,9 @@ def modifyModule(self, node, *args, **kwargs):
                 cb_key = b.name + ui_name
 
                 if cb_key in callbacks:
+                    if not self.combine_callbacks:
+                        raise ksp_ast.ParseException(b, "Callback already declared! (Combine Duplicate Callbacks is currently disabled)")
```

Comment:
Perhaps better wording that doesn't include parentheses: "This callback has already been declared! Either remove the duplicate, or enable the Combine Duplicate Callbacks option."